packages/site/src/layout/NavMenu.tsx

```diff
@@ -92,7 +92,9 @@ export const NavMenu = ({ mainContentRef }: NavMenuProps) => {
           >
             <Icon name="search" color="greyBlue" />
             <span className={styles.searchButtonText}>
-              <Typography textColor={"textSecondary"}>Search</Typography>
+              <Typography size={"base"} textColor={"textSecondary"}>
```

Comment:
Safari has userAgent styling for button text that this handles.

Reply:
![image](https://github.com/user-attachments/assets/c74f7c14-e585-4eed-8aaa-374a7de6f64d)